build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.0'
+        classpath 'com.android.tools.build:gradle:4.1.3'
```

Comment:
Why do you have change the version to a older one?

Reply:
I've got the following errror : This version of the Android Support plugin for IntelliJ IDEA (or Android Studio) cannot open this project, please retry with version 4.2 or newer

I tried to update everything to 4.2.1 first but it didn't work.
By searching for help on internet, I found that post https://github.com/MoIbrahim15/Android-Kotlin-MVI-CleanArchitecture/issues/3
It proposes downgrading gradle to an older version and it worked.